src/HostManager.cpp

```diff
@@ -214,8 +214,8 @@ void HostManager::postInterHostEvent( const Host * pHost, const TEvent & event )
     allValidHosts = afterSendingHost;
     allValidHosts.append( beforeSendingHost );
 
-    for( int j = 0, total = allValidHosts.size(); j < total; ++j ) {
-        hostList.at( allValidHosts.at( j ) )->raiseEvent( event );
+    for(int allValidHost : allValidHosts) {
```

Comment:
Ah, okay. Well it went from being super confusing to super clear, so I'm pretty happy. Actually I should rename it to ``validHost``, current autogenerated name is a little dodgy.